Sources/QueuesFluentDriver/JobModel.swift

```diff
@@ -1,55 +1,48 @@
 import Foundation
-import Fluent
+import FluentKit
 import Queues
 
 public enum QueuesFluentJobState: String, Codable, CaseIterable {
     /// Ready to be picked up for execution
     case pending
+    
+    /// In progress
     case processing
+    
     /// Executed, regardless if it was successful or not
     case completed
 }
 
-extension FieldKey {
-    static var queue: Self { "queue" }
-    static var data: Self { "data" }
-    static var state: Self { "state" }
-
-    static var runAt: Self { "run_at" }
-    static var updatedAt: Self { "updated_at" }
-    static var deletedAt: Self { "deleted_at" }
-}
-
-class JobModel: Model {
-    public required init() {}
-    
-    public static var schema = "_jobs_meta"
+final class JobModel: Model {
+    static let schema = "_jobs_meta"
     
     /// The unique Job ID
     @ID(custom: .id, generatedBy: .user)
     var id: String?
     
     /// The Job key
-    @Field(key: .queue)
+    @Field(key: "queue")
     var queue: String
     
     /// The current state of the Job
-    @Field(key: .state)
+    @Enum(key: "state")
     var state: QueuesFluentJobState
     
     /// Earliest date the job can run
-    @OptionalField(key: .runAt)
+    @OptionalField(key: "run_at")
     var runAtOrAfter: Date?
```

Comment:
Regarding my comment here https://github.com/m-barthelemy/vapor-queues-fluent-driver/issues/29 could we get rid of this property or `data.delayUntil` since they are the same thing?

Reply:
Unfortunately, I think at this point we're stuck with it for compatibility with existing users, even given the semver-major break 😕

Reply:
I believe the duplicate only exists in 3.0.0-beta.1 and **not** prior, if that changes your mind?

Reply:
Ah, you are correct, and indeed it does! I'll ditch the duplicate 👍